postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -0,0 +1,4753 @@
+/*
+ * psql - the PostgreSQL interactive terminal
+ *
+ * Copyright (c) 2000-2018, PostgreSQL Global Development Group
+ *
+ * src/bin/psql/command.c
+ */
+#include "postgres_fe.h"
+#include "commandbteq.h"
+
+#include <ctype.h>
+#include <time.h>
+#include <pwd.h>
+#ifndef WIN32
+#include <sys/stat.h>			/* for stat() */
+#include <fcntl.h>				/* open() flags */
+#include <unistd.h>				/* for geteuid(), getpid(), stat() */
+#else
+#include <win32.h>
+#include <io.h>
+#include <fcntl.h>
+#include <direct.h>
+#include <sys/stat.h>			/* for stat() */
+#endif
+
+#include "catalog/pg_class_d.h"
+#include "portability/instr_time.h"
+
+#include "libpq-fe.h"
+#include "pqexpbuffer.h"
+#include "fe_utils/string_utils.h"
+
+#include "common.h"
+#include "copy.h"
+#include "crosstabview.h"
+#include "describe.h"
+#include "help.h"
+#include "input.h"
+#include "large_obj.h"
+#include "mainloop.h"
+#include "fe_utils/print.h"
+#include "psqlscanslash.h"
+#include "settings.h"
+#include "variables.h"
+
+/*
+ * Editable database object types.
+ */
+typedef enum EditableObjectType
+{
+	EditableFunction,
+	EditableView
+} EditableObjectType;
+
+/* local function declarations */
+static dotResult exec_command(const char *cmd,
+			 PsqlScanState scan_state,
+			 ConditionalStack cstack,
+			 PQExpBuffer query_buf,
+			 PQExpBuffer previous_buf);
+static dotResult exec_command_a(PsqlScanState scan_state, bool active_branch);
```

Comment:
Please, set up your coding environment to not have problems with indentation / spaces (line 57-60 and others).

Reply:
Ok